lib/ncr_dispatcher.rb

```diff
@@ -0,0 +1,6 @@
+class NcrDispatcher < LinearDispatcher
```

Comment:
Maybe just add a comment saying this is a placeholder for doing per-user preferences, and linking to the story?